js/store.js

```diff
@@ -192,35 +205,26 @@ const store = {
     }
   },
 
-  getThirdParties(hostname) {
-    if (!hostname) {
-      throw new Error('getThirdParties requires a valid hostname argument');
-    }
-
-    const firstParty = this.getWebsite(hostname);
-    if ('thirdPartyHostnames' in firstParty) {
-      return firstParty.thirdPartyHostnames;
-    }
-
-    return null;
-  },
-
   setWebsite(hostname, data) {
     const websites = clone(this._websites);
```

Comment:
Are you referring to the clone of `this._websites` or the existence of `this._websites` itself? This method is based on the original method in terms of the clone. I could remove `websites` and modify `this._websites` directly -- otherwise are you suggesting to make setWebsite async and to await `this._write`?